src/client/game.cpp

```diff
@@ -4343,14 +4343,14 @@ void Game::showDeathFormspec()
 void Game::showPauseMenu()
 {
 #ifdef HAVE_TOUCHSCREENGUI
-	static const std::string control_text = strgettext("Default Controls:\n"
-		"No menu visible:\n"
-		"- single tap: button activate\n"
-		"- double tap: place/use\n"
+	static const std::string control_text = strgettext("Default controls:\n"
+		"No menu open:\n"
 		"- slide finger: look around\n"
-		"Menu/Inventory visible:\n"
+		"- tap: place/use\n"
+		"- long tap: dig/punch\n"
+		"Menu/inventory open:\n"
 		"- double tap (outside):\n"
```

Comment:
This might need to be changed as single tap is already "double tap".

Reply:
It looks like the code to close menus has its own, separate double-click detection implementation, which is not affected by my changes.

https://github.com/minetest/minetest/blob/d3a6ee00e63cc0a4adcaa7598ad5614f1e419515/src/gui/modalMenu.h#L93-L99

https://github.com/minetest/minetest/blob/d3a6ee00e63cc0a4adcaa7598ad5614f1e419515/src/gui/modalMenu.cpp#L115-L163

However, on my Android device, both on the `master` branch and with this PR, I can close menus (tested with the pause menu and the MTG inventory) with a single tap outside. But since this is already the case without this PR, I think it should be addressed in a different PR, as it could be quite complicated to figure out the intended behaviour.

Reply:
So, the behavior here is a bit funnier than I thought, but still not relevant to this PR. I've tested different menus. Some of them can be closed with a single tap, some only with a double tap. Which menus take how many taps to close is, as far as I have tested, consistent between `master` and this PR.

Reply:
I found out that, in some rare cases (can not reproduce yet), I need to tap outside twice to close a menu/inventory. So, I guess it is better to leave it as it is.

Reply:
> I found out that, in some rare cases (can not reproduce yet), I need to tap outside twice to close a menu/inventory. So, I guess it is better to leave it as it is.

That's approximately what I was trying to say with my two comments :)